index.html

```diff
@@ -260,6 +260,439 @@ <h1 class="primary-text1">
+                    </svg>
+                </a>
+                </li>
+            </ul>
+        </div>
+        <ul class="about-me-skill-list">
+            <li class="select">
+                    <!-- <select class="select">
+                      <option value="0">Languages</option>
+                      <option value="1">JavaScript</option>
+                      <option value="2">HTML</option>
+                      <option value="3">CSS</option>
+                    </select> -->
+                    <span class="select-text">Languages</span>
+                    <span class="select-svg"><svg width="8" height="15" viewBox="0 0 8 15" fill="none" xmlns="http://www.w3.org/2000/svg">
+                        <path fill-rule="evenodd" clip-rule="evenodd" d="M0.292893 0.439622C0.683417 0.0490974 1.31658 0.0490974 1.70711 0.439622L7.70711 6.43962C8.09763 6.83015 8.09763 7.46331 7.70711 7.85384L1.70711 13.8538C1.31658 14.2444 0.683417 14.2444 0.292893 13.8538C-0.0976311 13.4633 -0.0976311 12.8301 0.292893 12.4396L5.58579 7.14673L0.292893 1.85384C-0.0976311 1.46331 -0.0976311 0.830146 0.292893 0.439622Z" fill="#505F79"/>
+                        </svg>
+                    </span>
+            </li>
+            <svg width="327" height="2" viewBox="0 0 327 2" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <rect y="0.146729" width="327" height="1" fill="#DFE1E6"/>
+            </svg>
+                
+            <li class="frame-1 select">
+                <!-- <select class="select">
+                  <option value="0">Languages</option>
+                  <option value="1">JavaScript</option>
+                  <option value="2">HTML</option>
+                  <option value="3">CSS</option>
+                </select> -->
+                <span class="select-text">Frameworks</span>
+                <span class="select-svg"><svg width="8" height="15" viewBox="0 0 8 15" fill="none" xmlns="http://www.w3.org/2000/svg">
+                    <path fill-rule="evenodd" clip-rule="evenodd" d="M0.292893 0.439622C0.683417 0.0490974 1.31658 0.0490974 1.70711 0.439622L7.70711 6.43962C8.09763 6.83015 8.09763 7.46331 7.70711 7.85384L1.70711 13.8538C1.31658 14.2444 0.683417 14.2444 0.292893 13.8538C-0.0976311 13.4633 -0.0976311 12.8301 0.292893 12.4396L5.58579 7.14673L0.292893 1.85384C-0.0976311 1.46331 -0.0976311 0.830146 0.292893 0.439622Z" fill="#505F79"/>
+                    </svg>
+                </span>
+        </li>
+        <svg width="327" height="2" viewBox="0 0 327 2" fill="none" xmlns="http://www.w3.org/2000/svg">
+            <rect y="0.146729" width="327" height="1" fill="#DFE1E6"/>
+        </svg>
+        <li class="frame-1 select">
+            <!-- <select class="select">
+              <option value="0">Languages</option>
+              <option value="1">JavaScript</option>
+              <option value="2">HTML</option>
+              <option value="3">CSS</option>
+            </select> -->
```

Comment:
- [ ] Please do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history.
 - [ ] The 'Languages' item of this list should have the list of languages(e.g., "JavasScript", "HTML" and "CSS"), with their cards and an image attached to it below the 'Languages' item. Please add this since this is part of the Figma template.
 -Your design:
<img width="374" alt="image" src="https://user-images.githubusercontent.com/61919617/191733432-4c0f8493-ebbd-4c37-8947-e08ae6fd65ae.png">

 -Figma template:
<img width="339" alt="image" src="https://user-images.githubusercontent.com/61919617/191733478-ca919dcd-064f-4ebe-8b47-61674d9feec9.png">